### PR TITLE
tutorial intro tweaks

### PR DESCRIPTION
**Pull request by sryza**
### Summary & Motivation

A few tweaks to the tutorial intro because I'm very picky about how software-defined assets get talked about.

### CODE REVIEW

File: docs/content/tutorial/introduction.mdx [hunk 1]
```diff
@@ -23,9 +23,10 @@ An asset is an object in persistent storage that captures some understanding of
 
 If you have an existing data pipeline, you likely already have assets.
 
-Software-defined assets are Dagster's way of enabling data engineers to deliver value and control their data pipelines. Writing SDAs lets you focus on the contents of the data and less on other aspects like what it depends on, how it's stored, or when it needs to be refreshed.
+_Software-defined assets_ are a Dagster concept that allows you to write data pipelines in terms of the assets that they produce. How they work: you write code that describes an asset that you want to exist, along with any other assets that the asset is derived from, and a function that can be run to compute the contents of the asset.
+
+Here's an example. The asset is a dataset called "topstories", and it's computed by starting with an asset called "topstory_ids" and then fetching data for each of those IDs.
```
Comment (erinkcochran87) [inline docs/content/tutorial/introduction.mdx:28]:
```suggestion
Here's an example. The asset is a dataset called `topstories`, and it's computed by starting with an asset called `topstory_ids` and then fetching data for each of those IDs.
```

File: docs/content/tutorial/introduction.mdx [hunk 2]
```diff
@@ -42,7 +43,7 @@ def topstories(topstory_ids):
 
 ### Building a DAG of assets
 
-Defining multiple assets creates a DAG (directed acyclic graph) of what assets depend on other assets. This DAG helps you to:
+A set of assets naturally forms a [DAG (directed acyclic graph)](https://en.wikipedia.org/wiki/Directed_acyclic_graph), where the edges correspond to data dependencies between assets. This DAG helps you to:
```
Comment (erinkcochran87) [inline docs/content/tutorial/introduction.mdx:46]: I keep getting stuck on `naturally` here and I'm not sure why. Are there other things that could form a DAG? Could this be `A set of assets forms a DAG` and it's still accurate?

That being said, I'm not gonna block you on what is probably pretty nitpicky 😬

**Review by erinkcochran87 — APPROVED** [review]

File: docs/content/tutorial/introduction.mdx (hunk 2 above)
Comment (sryza) [reply docs/content/tutorial/introduction.mdx:46]: I think we can take out "naturally" without losing meaning

**Comment by tacastillo** [comment]
LGTM given Erin's feedback is added!

**Pull request merged**